src/main/kotlin/CarRacing.kt

```diff
@@ -0,0 +1,24 @@
+class CarRacing(
+    val carNumber: Int = 0,
+    val tryNumber: Int = 0,
+) {
+    fun start() {
+        println("자동차 대수는 몇 대인가요?")
+        val carNumber = readLine()!!.toInt()
+        println("시도할 회수는 몇 회인가요?")
+        val tryNumber = readLine()!!.toInt()
+
+        val cars = MutableList(carNumber) { Car() }
+        for (i in 0 until tryNumber) {
+            for (j in 0 until carNumber) {
+                cars[j].move()
+            }
```

Comment:
cars.for
```suggestion
           cars.forEach {
                it.move()
            }
```
다양한 코틀린의 확장함수를 이용해보아요!

Reply:
https://github.com/next-step/kotlin-racingcar/pull/1429/files#diff-248c33fad6bec744ec609db79ab3189406e700c271739395536b9682980d91ccR10 로 변경하였습니다